modules/users/server/config/users.server.config.js

```diff
@@ -11,7 +11,7 @@ var passport = require('passport'),
 /**
  * Module init function
  */
-module.exports = function (app, db) {
+module.exports = function (app) {
```

Comment:
We should note the potential breaking change in the CHANGELOG. 

Anyone merging from master should be aware of the changes, and it seems like it would be easy for them to add back the `db` in their exports where they rely on it. Adding `app.local.db` wouldn't be a difficult task either.

I'm a bit skeptical that many use cases would arise that would require someone to rely on accessing the `db` instance from server-side config/controllers.

I'm not opposed to adding `db` back to all the callbacks though.